backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -14,8 +14,8 @@
 
 package com.google.sps;
 
-import com.google.sps.data.Match;
-import com.google.sps.data.Participant;
+import main.java.com.google.sps.data.Match;
+import main.java.com.google.sps.data.Participant;
```

Comment:
here and elsewhere: please make sure all the package names remain as com.google.sps....

Reply:
On IntelliJ  it states that package names should correspond to the file path, in that case should all work be in a com.google.sps. package ?

Reply:
Or is this only for Janet's Matching algorithm files ?

Reply:
yeah, since this file is in sps the package is com.google.sps

a file in java/com/google/sps/data would have package com.google.sps.data

Reply:
Okay just to clarify, Currently our backend files are in `src/main/java/com/google/sps/ `folder name``  . I believe the issue is storing the files in the main folder instead of directly in src so a file in src/main/java/com/google/sps/data should be moved from this directory into a `src/com/google/sps/data` ?

Reply:
Or should I be ignoring IntelliJ ?

Reply:
Ignore intelliJ.

Reply:
Ack